csharp-basics/exercises/ClassesAndObjects/Account/Account.cs

```diff
@@ -2,22 +2,25 @@
 {
     class Account
     {
-        private string _name;
         private double _money;
+        public string Name { get; set; }
```

Comment:
Kāpēc public?

Reply:
Jo sākotnēji bija private string _name; un public string Name{get => _name; set=> _name = value}, līdz ar to _name property ir vienmēr iespējams mainīt, nomainot Name, es vienkārši noņēmu to extra soli un atstāju tikai public string Name, lai būtu iespēja gan setot, gan getot tā pat kā iepriekš